Sources/TIM/AppAuth/AppAuthController.swift

```diff
@@ -47,13 +47,19 @@ public final class AppAuthController: OpenIDConnectController {
 
     private func doAuthState(
         request: OIDAuthorizationRequest,
-        presentingViewController: UIViewController,
+        presentingViewController: UIViewController?,
         willPresentSafariViewController: @escaping (SFSafariViewController) -> Void,
         shouldAnimate: @escaping () -> Bool,
         didCancel: @escaping () -> Void,
         callback: @escaping (Result<OIDAuthState, TIMAuthError>) -> Void) {
+            
+            guard customOIDExternalUserAgent != nil || presentingViewController != nil else {
+                callback(.failure(.noUserAgent))
+                return
+            }
+            
             let externalUserAgent = customOIDExternalUserAgent ?? AuthSFController(
-                presentingViewController: presentingViewController,
+                presentingViewController: presentingViewController!,
```

Comment:
Thought line 56 to 59 was the most clear and self-explanatory way of writing it.
I know this is controversial with a force unwrap, but we are basically making the check, to ensure this value will never be nil in this case.